### DISCUSSION TITLE
issue with one of the two poe coordinators I just received

### DISCUSSION THREAD

**Discussion by YAMLcase** [tube0013/tube_gateways#119] [General]
Hi @tube0013, I just received both tubeszb-cc2652-poe-2022 and getting them setup.  One works perfectly, the other will not allow any devices to join.  here's what I've tried:

- flashed ESPHome via USB and zigbee module following your [Instructions here](https://github.com/tube0013/tube_gateways/tree/main/models/current/tubeszb-cc2652-poe-2022#cc2652-firmware-flashing)
- Tried several different locations, ports on the PoE switch (including the known good port the other coordinator is using)
- devices that I fail to connect to the suspect coordinator will join fine to the other coordinator
- reseated the cc2652 module and the antenna
- Swapped antenna

I can see Z2M connecting fine and getting info back from coordinator.  Also on the device's web server there's a `new client connected from <IP>` message when restarting Z2M.

In my setup I have 2 zigbee2MQTT instances running unmodified koenkk/zigbee2mqtt:1.29.2 images.

Baffled

**Comment by tube0013** [comment]
Hi,

Sorry about this. I test all coords before going out. Can you try an energy-scan on the one that isn't working well:

https://github.com/zigpy/zigpy-cli
Specifically:
https://github.com/zigpy/zigpy-cli#performing-an-energy-scan

the command will be:

`zigpy radio znp socket://IP_ADDRESS:6638 energy-scan`

You might want to do it with both and compare the results you should see similar.

**Reply by YAMLcase** [reply]
Here's the scan results:

Bad coordinator:
```
Channel energy (mean of 4 / 5):
------------------------------------------------
 + Lower energy is better
 + Active Zigbee networks on a channel may still cause congestion
 + TX on 26 in North America may be with lower power due to regulations
 + Zigbee channels 15, 20, 25 fall between WiFi channels 1, 6, 11
 + Some Zigbee devices only join networks on channels 15, 20, and 25
 + Current channel is enclosed in [square brackets]
------------------------------------------------
 -  11       0.00%
 -  12       0.00%
 -  13       0.00%
 -  14       0.00%
 -  15       0.00%
 -  16       0.00%
 -  17       0.00%
 -  18       0.00%
 -  19       0.00%
 -  20       0.00%
 -  21       0.00%
 -  22       0.00%
 -  23       0.00%
 - [24 ]     0.00%
 -  25       0.00%
 -  26*      0.00%
 ```

Good coordinator:

```
Channel energy (mean of 4 / 5):                                           
------------------------------------------------                          
 + Lower energy is better                                                 
 + Active Zigbee networks on a channel may still cause congestion         
 + TX on 26 in North America may be with lower power due to regulations   
 + Zigbee channels 15, 20, 25 fall between WiFi channels 1, 6, 11         
 + Some Zigbee devices only join networks on channels 15, 20, and 25      
 + Current channel is enclosed in [square brackets]                       
------------------------------------------------                          
 -  11       0.00%                                                        
 -  12       0.00%                                                        
 -  13      18.53%  ##################                                    
 -  14       0.00%                                                        
 -  15      25.88%  #########################                             
 -  16      25.49%  #########################                             
 -  17      35.88%  ###################################                   
 -  18      43.04%  ###########################################           
 -  19      24.90%  ########################                              
 -  20      22.65%  ######################                                
 -  21       0.00%                                                        
 -  22      13.92%  #############                                         
 - [23 ]    10.29%  ##########
 -  24      10.00%  ##########
 -  25       9.22%  #########
 -  26*     43.73%  ###########################################                                    
```

**Comment by tube0013** [comment]
well, I must of missed this, as I run this is the test I run! can you reach out to me at store @ tubeszb.com with your name and order number and I'll get a replacement zigbee pcb that you can swap into place on the poe board out to you.

**Reply by YAMLcase** [reply]
will do, thanks for great customer service!

**Reply by tube0013** [reply]
The replacement went out today. I came across another that does the same thing so I will be reaching out to manufacturer of the modules tomorrow night. Figured it out too...those 2 pads are not supposed to be empty and break the rf path 🤦‍♂️ ![image](https://user-images.githubusercontent.com/11084412/215297640-4fa68023-48c5-4766-8571-e52fe893284a.jpeg)

**Reply by YAMLcase** [reply]
What's supposed to go there?

**Reply by tube0013** [reply]
Another one of those tiny 01005 resisters. I took one from a bad module and repaired the one I had to prove it out. I suspect bridging it would suffice too. 

Dealing with those components by hand is not easy - I nudged other components out of place while trying to bridge it and ultimate place one there and had to move them back all while it was on a hot plate.

**Reply by tube0013** [reply]
FWIW The one that got to you should have never gone out. That's my mistake so apologies for the inconvenience and appreciate the understanding.

**Comment by YAMLcase** [comment]
@tube0013 I received the replacement, looks better!

```
Channel energy (mean of 5 / 5):
------------------------------------------------
 + Lower energy is better
 + Active Zigbee networks on a channel may still cause congestion
 + TX on 26 in North America may be with lower power due to regulations
 + Zigbee channels 15, 20, 25 fall between WiFi channels 1, 6, 11
 + Some Zigbee devices only join networks on channels 15, 20, and 25
 + Current channel is enclosed in [square brackets]
------------------------------------------------
 -  11       0.00%
 -  12       0.00%
 -  13       2.27%  ##
 -  14       3.69%  ###
 -  15      18.43%  ##################
 -  16      24.24%  ########################
 -  17      44.00%  ############################################
 -  18      38.04%  ######################################
 -  19      14.75%  ##############
 -  20       7.69%  #######
 -  21       0.55%
 -  22       0.55%
 -  23       5.65%  #####
 - [24 ]     0.00%
 -  25       2.51%  ##
 -  26*     16.16%  ################
```

I'll put the defective one in the mail tomorrow

**Reply by tube0013** [reply]
Great, thank you
